src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,155 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String INDEX = "Index: ";
+    private static final String SIZE = ", Size: ";
+    private static final int NON_NEGATIVE_INDEX = 0;
```

Comment:
It would be better to find out better name for this constatnt couse every number biger than 0 is non negative